### PR TITLE
copy editing changes to summary

### PR DESCRIPTION
**Pull request by deepthi**
## Description
Wordsmith the release summary before code freeze.

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [x] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on CI?
-   [x] Documentation was added or is not required

### CODE REVIEW

File: changelog/21.0/21.0.0/summary.md [hunk 1]
```diff
@@ -136,18 +133,18 @@ preferentially routing to tablets in the same cell as the VTGate.
...
 `query_timeout` session system variable, or `query-timeout` flag is set.
-The order of precedence is: `QUERY_TIMEOUT_MS` > `query_timeout` > `query-timeout`.
+The order of precedence is: comment directive > session variable > VTGate flag.
 VTTablet overrides its default query timeout with the value received from VTGate.
 All timeouts are specified in milliseconds.
 
-When a query is executed inside a transaction, this behavior does not apply; instead,
-the smaller of the transaction timeout or the query timeout from VTGate is used.
+When a query is executed inside a transaction, there is an additional nuance. The actual timeout used will be the smaller 
+of the transaction timeout and the query timeout.
```
Comment (deepthi) [inline changelog/21.0/21.0.0/summary.md:147]: @harshit-gangal / @GuptaManan100 is this correct? That is, we pick one value from comment/session var/vtgate flag, and the other value from the transaction timeout, and apply the lower of the two.

File: changelog/21.0/21.0.0/summary.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 - **[Major Changes](#major-changes)**
     - **[Deprecations and Deletions](#deprecations-and-deletions)**
+        - [Deprecated VTTablet Flags](#vttablet-flags)
         - [Deletion of deprecated metrics](#metric-deletion)
-        - [VTTablet Flags](#vttablet-flags)
-        - [Metrics](#deprecations-metrics)
+        - [Deprecated Metrics](#deprecations-metrics)
```
Comment (frouioui) [inline changelog/21.0/21.0.0/summary.md:9]: It seems like we are using case, between `Deletion of deprecated metrics` and the other items (i.e. `Deprecated VTTablet Flags`)

**Review by frouioui — COMMENTED** [review]
One comment, otherwise it looks good to me

**Review by rohit-nayak-ps — APPROVED** [review]

File: changelog/21.0/21.0.0/summary.md (hunk 2 above)
Comment (deepthi) [reply changelog/21.0/21.0.0/summary.md:9]: Good point, but I don't think it's very important.

File: changelog/21.0/21.0.0/summary.md (hunk 1 above)
Comment (GuptaManan100) [reply changelog/21.0/21.0.0/summary.md:147]: Yes - 
```go
timeout := tsv.loadQueryTimeoutWithOptions(options)

	if txID == 0 {
		return timeout
	}

	// fetch the transaction timeout.
	txTimeout := tsv.config.TxTimeoutForWorkload(querypb.ExecuteOptions_OLTP)

	// Use the smaller of the two values (0 means infinity).
	return smallerTimeout(timeout, txTimeout)
```

**Review by frouioui — APPROVED** [review]

**Review by GuptaManan100 — APPROVED** [review]

**Pull request merged**